style.css

```diff
@@ -113,7 +113,7 @@ strong {
 
 b,
 strong {
-	font-weight: bolder;
+	font-weight: 900;
```

Comment:
I think this should remain `bolder`

In a more broader scenario I think the WordPress CSS Coding Standards should allow the use of `bolder` and `lighter` relative weights